feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -849,7 +852,11 @@ class BillingWrapper(
                     // only setObfuscatedAccountId for non-upgrade/downgrades until google issue is fixed:
                     // https://issuetracker.google.com/issues/155005449
                     replaceSkuInfo?.let {
-                        setUpgradeInfo(it)
+                        setSubscriptionUpdateParams(
+                            SubscriptionUpdateParams.newBuilder()
+                            .setOldPurchaseToken(replaceSkuInfo.oldPurchase.purchaseToken)
```

Comment:
Should we be using `it` on this line? 🤔  

```suggestion
                            .setOldPurchaseToken(it.oldPurchase.purchaseToken)
```